### PR TITLE
COA changes for BCOMP

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity#4226

*Description of changes:*
Corrected the templates and added the missing elements
Basic framework changes to get this working

Note: There are major changes in report.py coming up in the next PR. 
- Loading different templates for different entities
- Having common components
-Certification template to consider legal type
-Cleaning up and organizing report.py

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Comment by lekshmimallika-aot** [comment]
The sonar check code duplication error will be resolved in the next PR where style is going to be common

File: legal-api/report-templates/template-parts/bc-address-change/addresses.html [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
       <tr>
         <td class="delivery-address">
-          <div class="section-sub-title">Delivery Address</div>
-          <div>{{ bcAddressChange.offices.registeredOffice.deliveryAddress.streetAddress }}</div>
-          <div>{{ bcAddressChange.offices.registeredOffice.deliveryAddress.streetAddressAdditional }}</div>
-          <div>{{ bcAddressChange.offices.registeredOffice.deliveryAddress.addressCity }}
-            {{ bcAddressChange.offices.registeredOffice.deliveryAddress.addressRegion }}
-            &nbsp;{{ bcAddressChange.offices.registeredOffice.deliveryAddress.postalCode }}
+          <div class="section-sub-title">Delivery Address  {% if 'addressChanged' in registeredOfficeAddress.deliveryAddress.actions %}{{ labelNew() }}{% endif %}</div>
+          <div> {{ registeredOfficeAddress.deliveryAddress.streetAddress }}</div>
```
Comment (severinbeauvais) [inline legal-api/report-templates/template-parts/bc-address-change/addresses.html:11]: This is probably a non-breaking space anyway, but why did you add it for all address lines?

**Comment by severinbeauvais** [comment]
@kialj876 Is it a quick to get unit test "test_post_colin_filing" passing?

**Comment by lekshmimallika-aot** [comment]
> @kialj876 Is it a quick to get unit test "test_post_colin_filing" passing?

Will check that

**Comment by lekshmimallika-aot** [comment]
> Looks good.
> If I understand correctly, these codes are still not in use (except for style folder change), since the new template is not called yet.
> As I working on the new BC COD output, I will make minimum changes to report.py, knowing that you are refactoring it.

COD changes will be templates + report.py (mostly on the template parts section). The chip style needed is already added in style.html

File: legal-api/report-templates/template-parts/bc-address-change/addresses.html (hunk 1 above)
Comment (lekshmimallika-aot) [reply legal-api/report-templates/template-parts/bc-address-change/addresses.html:11]: I did a search and replace and I think extra space got added in the beginning on the replace text. Will remove that

**Comment by severinbeauvais** [comment]
> > Looks good.
> > If I understand correctly, these codes are still not in use (except for style folder change), since the new template is not called yet.
> > As I working on the new BC COD output, I will make minimum changes to report.py, knowing that you are refactoring it.
> 
> COD changes will be templates + report.py (mostly on the template parts section). The chip style needed is already added in style.html

Yup, that's where I'm at atm.

**Comment by lekshmimallika-aot** [comment]
> @kialj876 Is it a quick to get unit test "test_post_colin_filing" passing?

Fixed
